specification/gp-connect-prescriptions-management-fhir.yaml

```diff
@@ -549,73 +550,147 @@ paths:
+                          code: 3113111000001106
+                          display: Pulmicort 100 Turbohaler (AstraZeneca UK Ltd)
+                    subject:
+                      reference: Patient/9000000009
+                      display: Jane Smith
+                    effectivePeriod: 
+                      start: 2022-10-23T13:50:00+00:00
+                      end: 2023-01-10T13:50:00+00:00
+                    dateAsserted: 2020-01-12
+                    basedOn: 
+                      reference: MedicationRequest/0ae6ac98-6b0f-404f-a95b-c38a4318f686
+                      display: Continuous long term (repeat dispensing) 
+  /FHIR/R4/MedicationRequest:
+    get:
+      operationId: get-medicationrequest
+      summary: Get a patients medication requests
+      description: > 
+        Use this endpoint in order to get specific MedicationRequests referenced within the MedicationStatements.
+        
+        The request must include the following parameter:
+
+        * identifier
+      parameters:
+        - $ref: "#/components/parameters/BearerAuthorization"
+        - $ref: "#/components/parameters/CorrelationID"
+        - $ref: "#/components/parameters/RequestID" 
+        - in: query
+          name: patient:identifier
+          description: >
+            The NHS number of the patient whose requests you are searching for
+          required: false
+          schema:
+            type: string
+            example: '9000000009'
+      responses:
+        '200':
+          description: Successful request to view medication
+          content: 
+            application/fhir+json:
+              schema: 
+                type: object
+              examples:
+                example:
+                  value:
+                    resourceType: Bundle
+                    id: 2b62ff26-25a9-4a88-8cb6-f69e85fb6159
+                    identifier: 
+                      system: "https://tools.ietf.org/html/rfc4122"
+                      value: 86af4f6c-30c5-4947-acc7-2f55145181de
+                    type: searchset
+                    entry:
                       - resource: 
                           resourceType: MedicationRequest
-                          id: 7e68abae-a50a-4dd2-8445-7a2aa9936bee
-                          meta: 
-                            - profile: https://fhir.hl7.org.uk/StructureDefinition/UKCore-MedicationRequest
-                          status: completed
-                          intent: plan
-                          medicationReference:
-                            reference: 8b67891-df9n-3nfu-jnvc8-18s08fvhs0
-                          subject: 
-                            reference:  Patient/9000000009
-                          authoredOn: 2018-01-06
-                          requester: 
-                            reference:  Patient/9000000009
-                          substitution: false
+                          id: b269d1d7-1acf-47bb-8b3c-e38b583d9a07
```

Comment:
the really important ID to include is the EPS prescription ID. In EPS this uses 
```
"identifier" : {
          "system" : "https://fhir.nhs.uk/Id/prescription-order-number",
          "value" : "CDF34E-A99968-4FF3BQ"
        }
```
Note the ID format which is UUID-like but not actualy a UUID.
Think we should say 'EPS R2 prescriptions will include EPS prescription ID which can be used to track the status of the prescription'.